### ISSUE TITLE
Accordion: Implement attribute "aria-hidden" and other missing wai-aria properties

### ISSUE THREAD

**Issue by leifoolsen** [leifoolsen/mdl-ext#10]
An accordion should convey the visibility of each tabpanel by maintaining its aria-hidden state.

**Comment by leifoolsen** [comment]
An accordion should have the following properties:

* The accordion component must have a role of tablist and have aria-multiselectable="true" This will enable an assistive technology, such as screen reader, to convey that the tablist is an accordion or a multi selectable tablist. This will also tell the user that the keyboard navigation matches an accordion and not a tablist.
* Each header tab in the tablist has a role of tab.
* The accordion panel uses the role tabpanel and should have an aria-labelledby relationship referencing the corresponding header having a role of tab
* An accordion should manage the expanded/collapsed state of each tab by maintain its aria-expanded state.
* An accordion should manage the selected state of each tab by maintaining its aria-selected state.
* An accordion should convey the visibility of each tabpanel by maintaining its aria-hidden state.

Not part of the definition, but should be implemented
* When an accordion panel is disabled, aria-disabled is set to true.
